lib/lib_rss.php

```diff
@@ -397,12 +397,13 @@ function is_referer_from_same_domain() {
  */
 function check_install_php() {
 	$pdo_mysql = extension_loaded('pdo_mysql');
+	$pdo_pgsql = extension_loaded('pdo_pgsql');
 	$pdo_sqlite = extension_loaded('pdo_sqlite');
 	return array(
 		'php' => version_compare(PHP_VERSION, '5.3.8') >= 0,
 		'minz' => file_exists(LIB_PATH . '/Minz'),
 		'curl' => extension_loaded('curl'),
-		'pdo' => $pdo_mysql || $pdo_sqlite,
+		'pdo' => $pdo_mysql || $pdo_sqlite || $pdo_pgsql,
```

Comment:
This one's still at the end. :-P

Reply:
In the rest of the code, there are many places with that order, for historical reasons (MySQL was used first, and then SQLite compatibilité, and last PostgreSQL). So I do not mind either way :-)

Reply:
It's one of those things where on the one hand it seems like an enormous nitpick. On the other hand I've learned through bitter experience how innocent code where different people just randomly add some stuff at the end or bottom can become an unparseable blob before you good and well realize what happened.

You could quite validly point out that the set will always remain minuscule, but it still serves as an example that might be followed.

Anyway, that's just a braindump. I already explicitly marked it approved after all. ;-)